### PR TITLE
Editing settings and configuration

### PR DESCRIPTION
**Pull request by DelazJ**
The digitizing environment has evolved in the last releases but it looks like the doc does not really reflect these changes.
This PR is the first step of the refactoring/updating of all this. It focuses on the settings section:
* add some indices to build the index list
* precise some options behaviour
* add tips and note

Review (especially for english) is very welcome

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -30,14 +31,17 @@ SpatiaLite, PostGIS, MSSQL Spatial and Oracle Spatial vector layers and tables.
...
 
-Before we can edit vertices, we must set the snapping tolerance and search
-radius to a value that allows us an optimal editing of the vector layer
-geometries.
+For an optimal and accurate edit of the vector layer geometries, we need to set
+an appropriate value of snapping tolerance and search radius for features vertices.
+
+.. % FIXME Different menus are used below to identify "Options"/"Snapping Options" sub-menu (Settings, Edit, File). Would be nice to check what is correct according to OS.
+   File menu no longer exists if I'm not wrong (replaced by Project). On Windows, all is in the Settings menu. What about other platforms? Inputs are welcome.
+
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: comments are welcome here

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 2]
```diff
@@ -56,20 +60,26 @@ The snapping tolerance setting affects all tools that work with tolerance.
    tolerance doesn't have to be changed after zoom operations. In our small
    digitizing project (working with the Alaska dataset), we define the
    snapping units in feet. Your results may vary, but something on the order
-   of 300 ft at a scale of 1:10000 should be a reasonable
-   setting.
-#. A layer-based snapping tolerance can be defined by choosing
+   of 300 ft at a scale of 1:10000 should be a reasonable setting.
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: feet? Why don't we use meter as ISU?

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 3]
```diff
@@ -56,20 +60,26 @@ The snapping tolerance setting affects all tools that work with tolerance.
...
+#. A layer-based snapping tolerance that overrides the global snapping option
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: option**s**?

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 1 above)
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: KDE: "Snapping Options" is in "Settings" menu. But I think this is the same for all OS (not sure though)

*Project settings* menu is different : For WIN and Gnome it is in *Project* menu, For KDE, it is in *Options* menu.

**Comment by yjacolin** [comment]
LGTM

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 2 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: I don't know and it does not really disturb me given that the example is from Alaska QGIS project wich is maybe in a feet units (didn't check)

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: does it mean that in KDE, we have:
* Settings menu  --> Snapping options (what other sub-menus are along this one
* and Options menu --> Project settings?
And how do you open the `Options` dialog (general, digitizing, variable...tabs)  which is under Settings menu (in Windows)?

Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: Sorry I mixed english and french menu name. Options = Settings in my last comment.

Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: So in KDE, under Settings menu, you have `Options`, `Snapping Options`, `Customization`...
The only difference is that KDE offers also `Project Settings`. Right?

Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: and *style manager*! I wondered if there is not already a place describing this difference somewhere in the doc.

Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: And also `Custom CRS`?
Yes, description in http://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_gui.html#help though I think I broke the list in a previous PR due to a misinterpretation of:
> Please note that for Linux nix, the menu bar items listed above are the default ones in the KDE window manager. In GNOME, the Settings menu has different content and its items have to be found here:

AFAICS, Gnome menus are the same as Windows and Mac (at least in Settings menu) so the norme should be a description of how those OSes display (the most common states) and then indicate the special situation of Kde. and more globally list in the table what are most present in the different OS and a text on differences later.

Furthermore, I think that the differences should be pointed at the end of the concerned section (menu) description and not at the end of "all" menus.
worth another PR :)

**Comment by yjacolin** [comment]
Could you merge it?

**Comment by DelazJ** [comment]
Have to clean my comment L37 before. And will do it.

**Pull request merged**